### PR TITLE
Interactive units

### PR DESCRIPTION
**Pull request by sudarsan-surendralal**
Ensures that unit conversions are proper even for interactive jobs

### CODE REVIEW

File: pyiron_atomistics/lammps/base.py [hunk 1]
```diff
@@ -61,6 +61,28 @@ def __init__(self, project, job_name):
...
+        """
+        if self.input.control["units"] is not None:
+            return self.input.control["units"]
+        else:
+            # Default to metal units
+            return "metal"
+
+    @units.setter
+    def units(self, val):
+        allowed_types = ['metal', 'real', 'si', 'cgs', 'lj']
```
Comment (jan-janssen) [inline pyiron_atomistics/lammps/base.py]: Having the static list defined here is dangerous, as we might forget to update it later. Can you use `LAMMPS_UNIT_CONVERSIONS.keys()` ?

File: notebooks/integration/interactive_units.ipynb [hunk 2]
```diff
@@ -0,0 +1,161 @@
+{
```
Comment (jan-janssen) [inline notebooks/integration/interactive_units.ipynb:1]: An explanation in the notebook would be nice.

Comment (sudarsan-surendralal) [reply notebooks/integration/interactive_units.ipynb:1]: I added some text in the beginning of the notebook!

**Comment by sudarsan-surendralal** [comment]
@jan-janssen shall I merge this for now?

**Pull request merged**